### DISCUSSION TITLE
Error I don't know how to fix

### DISCUSSION THREAD

**Discussion by rm-ccu** [TomSchimansky/CustomTkinter#878] [Q&A]
Got this error while making a GUI, I believe it is caused by the fact that I call 2 functions to open dialog boxes and it errors out for some reason. Anyone know a fix for this?

![image](https://user-images.githubusercontent.com/73251739/208212332-db4a1db4-9a28-48d5-ae54-36a75c676331.png)

**Comment by Akascape** [comment]
@InkEffect Maybe because you defined the same variable `nkB` two times.

**Reply by rm-ccu** [reply]
Thanks, that was a problem but not the problem that I needed to solve.

![image](https://user-images.githubusercontent.com/73251739/208269106-266a7ab3-b548-46a9-aea1-6a0843d20e3f.png)
